loadgen/scenario.go

```diff
@@ -95,6 +95,8 @@ type ScenarioInfo struct {
 	Namespace string
 	// Path to the root of the omes dir
 	RootPath string
+	// Optional callback to receive status updates from the scenario.
+	StatusCallback func(any)
```

Comment:
A little more detail in this comment would be good. Explain that the callback is called with info specific to the scenario, and the person defining the callback needs to know what that type is, etc